pkgs/development/libraries/mlt/default.nix

```diff
@@ -36,6 +36,8 @@
 , enableSDL2 ? true
 , SDL2
 , gitUpdater
+, enableGlaxnimate ? true
```

Comment:
Does this need to be optional here?

Reply:
Since the library creates a compilation flag to enable this or not, I think it makes sense to make it optional here… But to be honest I don't care so much.

Reply:
Not every optional dependency should be a compilation flag, or we'll create an unmaintainable combinatoric explosion of possible flags.

Reply:
Alright, I don't mind to remove it then. I saw that you plan to work on it now, should I let you do it?

Reply:
I'll remove it in the 24.05.2 PR.

Reply:
Ok thanks a lot!